src/nvim/ex_docmd.c

```diff
@@ -1519,6 +1576,169 @@ bool parse_cmdline(char *cmdline, exarg_T *eap, CmdParseInfo *cmdinfo, char **er
   return true;
 }
 
+// Get end of an Ex-command argument with zero-indexed number, discarding trailing whitespace.
```

Comment:
Nit: no dash in "Ex-command".

Reply:
<img width="262" alt="image" src="https://user-images.githubusercontent.com/5308024/167308826-b1d43089-b195-4f1c-87ed-c7873dfd400c.png">